### ISSUE TITLE
Make "Elements List" palette discoverable without needing to know the keyboard shortcut

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#14720]
NVDA+f7 is "occult knowledge"!

Fine to have a shortcut like this, but there should be a way to discover this important UI feature without consulting the docs.

### Describe the solution you'd like
An obvious place for "Elements List" to be discovered is in the taskbar menu, perhaps under "Tools" so that the palette can be opened from there. However, I have been informed that once you are messing around in the menu, the focus (e.g. on a web page or some other app) is lost. 

I would hope there is some other mechanism that could make this useful feature more obvious.

This would be especially useful for frontend developers, who are very unlikely to be aware that the palette exists.

(I suggest this general design principle: If NVDA has GUI parts, there should be a way to open those parts via the GUI).

**Comment by jcsteh** [comment]
It's worth noting that most other features in browse mode are only available with shortcut keys; e.g. quick navigation, browse/focus mode switching, etc. In this sense, the argument concerning discoverability by front-end developers doesn't quite hold. By that logic, we'd also need to make quick navigation, mode switching, etc. available via the NVDA menu, but there's no way that can be done.

This would also be somewhat tricky to implement because Elements List is only available in specific contexts. When the menu opens, focus has already moved away from that context (out of the browser and into the menu). It's not impossible, but the juice might not be worth the squeeze. Ultimately, that's not my decision, but I thought it worth noting.

**Comment by brennanyoung** [comment]
Thanks! That's useful. I had not appreciated that the Elements List GUI is hooked up to a given target in a way that Speech Viewer is not. I will change the title of the issue to reflect this.

**Comment by seanbudd** [comment]
Closing this as can't fix as per @jcsteh comment
